src/Hook/Navigation.tsx

```diff
@@ -0,0 +1,17 @@
+export const useNavigation = (navigation: any) => {
```

Comment:
I think it's not a good idea to have a hook folder. It's better to architect things by their domain. Here it's the fact to go from page to page, so you can just define folder `navigation` or `router`, and name this file `useNavigation`.

> navigation
    > useNavigation.ts

Reply:
And keep uppercase for components